### PR TITLE
Fix kafka tests

### PR DESCRIPTION
**Pull request by mar-kolya**

### CODE REVIEW

File: dd-java-agent/instrumentation/kafka-clients-0.11/kafka-clients-0.11.gradle [hunk 1]
```diff
@@ -31,7 +31,9 @@ dependencies {
   // This seems to help with jar compatibility hell.
   latestDepTestCompile group: 'org.apache.kafka', name: 'kafka_2.11', version: '+'
   latestDepTestCompile group: 'org.apache.kafka', name: 'kafka-clients', version: '+'
-  latestDepTestCompile group: 'org.springframework.kafka', name: 'spring-kafka', version: '+'
-  latestDepTestCompile group: 'org.springframework.kafka', name: 'spring-kafka-test', version: '+'
+  // FIXME: kafka-spring 2.3 has breaking API changes, we need to adds tests for it at some point,
```
Comment (jbachorik) [inline dd-java-agent/instrumentation/kafka-clients-0.11/kafka-clients-0.11.gradle]: A typo - `need to adds` -> `need to add` (same for the other file)

Comment (mar-kolya) [reply dd-java-agent/instrumentation/kafka-clients-0.11/kafka-clients-0.11.gradle]: Thanks!

**Comment by mar-kolya** [comment]
> Is the change to 2577 files in integrations-core submodule really a part of this PR?

Good catch. I've removed that now, but things might still be failing... :(

**Comment by randomanderson** [comment]
This exact pull request already exists with https://github.com/DataDog/dd-trace-java/pull/1023

**Comment by mar-kolya** [comment]
Yeah, sorry, I only noticed that after I've opened this one

**Comment by mar-kolya** [comment]
@randomanderson that being said I think all other failures are actually due to unrelated instabilities in tests and this can be safely merged because it actually solves (at least temporarily) an underlying issue.

**Comment by dougqh** [comment]
@mar-kolya @randomanderson I'm fine with going ahead and merging either pull request if we think that's okay.  I've been monitoring mine to get it through CI this morning.

**Pull request closed without merging**